prov/mrail/src/mrail.h

```diff
@@ -212,9 +212,6 @@ struct mrail_ep {
 	struct util_buf_pool	*req_pool;
 	struct util_buf_pool 	*ooo_recv_pool;
 	struct slist		deferred_reqs;
-
-	ofi_fastlock_acquire_t	lock_acquire;
-	ofi_fastlock_release_t	lock_release;
```

Comment:
The commit message doesn't describe why this change is needed.  Please elaborate